easybuild/easyconfigs/s/Siesta/Siesta-3.2-pl-5-intel-2016.02-GCC-4.9.eb

```diff
@@ -0,0 +1,65 @@
+ structure calculations and ab initio molecular dynamics simulations of molecules and solids."""
+
+toolchain = {'name': 'intel', 'version': '2016.02-GCC-4.9'}
+toolchainopts = {'usempi': True}
+
+sources = [SOURCELOWER_TGZ]
+source_urls = ['https://dl.dropbox.com/u/20267285/SIESTA-DOWNLOADS/']
+
+# configure is in Src but building in Src is not allowed
+prebuildopts = 'cd Obj && ../Src/obj_setup.sh && ../Src/configure --enable-mpi && '
+
+# build siesta (must override COMP_LIBS in order to ensure usage of MKL's lapack instead of internal)
+buildopts = 'COMP_LIBS="" BLAS_LIBS="$LIBBLAS" LAPACK_LIBS="$LIBLAPACK" BLACS_LIBS="$LIBBLACS" '
+buildopts += 'SCALAPACK_LIBS="$LIBSCALAPACK" && '
+
+# build transiesta
+buildopts += 'cd .. && mkdir Obj2 && cd Obj2 && ../Src/obj_setup.sh && ../Src/configure --enable-mpi && '
+buildopts += 'make transiesta COMP_LIBS="" BLAS_LIBS="$LIBBLAS" LAPACK_LIBS="$LIBLAPACK" '
+buildopts += 'BLACS_LIBS="$LIBBLACS" SCALAPACK_LIBS="$LIBSCALAPACK" && '
+
+# build all utils
+buildopts += 'cd ../Util && sh ./build_all.sh'
+
+parallel = 1
+
+files_to_copy = [(['Obj/siesta', 'Obj2/transiesta', 'Util/Bands/eigfat2plot', 'Util/Bands/new.gnubands',
+                   'Util/Bands/gnubands', 'Util/CMLComp/ccViz', 'Util/COOP/mprop', 'Util/COOP/dm_creator',
+                   'Util/COOP/fat', 'Util/Contrib/APostnikov/eig2bxsf', 'Util/Contrib/APostnikov/xv2xsf',
+                   'Util/Contrib/APostnikov/md2axsf', 'Util/Contrib/APostnikov/rho2xsf',
+                   'Util/Contrib/APostnikov/vib2xsf', 'Util/Contrib/APostnikov/fmpdos',
+                   'Util/Denchar/Examples/2dplot.py', 'Util/Denchar/Examples/surf.py', 'Util/Denchar/Src/denchar',
+                   'Util/DensityMatrix/dm2cdf', 'Util/DensityMatrix/cdf2dm', 'Util/Eig2DOS/Eig2DOS',
+                   'Util/Gen-basis/ionplot.sh', 'Util/Gen-basis/gen-basis', 'Util/Gen-basis/ioncat',
+                   'Util/Grid/grid2cdf', 'Util/Grid/cdf2xsf', 'Util/Grid/cdf2grid', 'Util/Grid/grid2val',
+                   'Util/Grid/grid2cube', 'Util/HSX/hsx2hs', 'Util/HSX/hs2hsx', 'Util/Helpers/get_chem_labels',
+                   'Util/MPI_test/pi3', 'Util/Macroave/Src/macroave', 'Util/ON/lwf2cdf', 'Util/Optimizer/swarm',
+                   'Util/Optimizer/simplex', 'Util/Projections/orbmol_proj', 'Util/STM/simple-stm/plstm',
+                   'Util/SiestaSubroutine/FmixMD/Src/simple', 'Util/SiestaSubroutine/FmixMD/Src/driver',
+                   'Util/SiestaSubroutine/FmixMD/Src/para', 'Util/TBTrans/tbtrans', 'Util/VCA/mixps',
+                   'Util/VCA/fractional', 'Util/Vibra/Src/fcbuild', 'Util/Vibra/Src/vibrator', 'Util/WFS/readwf',
+                   'Util/WFS/readwfx', 'Util/WFS/info_wfsx', 'Util/WFS/wfs2wfsx', 'Util/WFS/wfsx2wfs',
+                   'Util/WFS/wfsnc2wfsx', 'Util/pdosxml/pdosxml', 'Util/pseudo-xml/xml2psf',
+                   'Util/test-xml/test-xml'], 'bin')]
```

Comment:
oh boy, what a mess... :-/

I guess these binaries are mixed with source and `.o` files?